### PR TITLE
overview and summary api

### PR DESCRIPTION
**Pull request by ankitmashu**
- [x] Docs Updated
- [x] Api Completed
- [x] Postman Updated
- [x] Test Cases

### CODE REVIEW

**Review by gopal-mahajan — CHANGES_REQUESTED** [review]
Please check the comments

File: src/main/java/iudx/rs/proxy/metering/util/DateValidation.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    long zonedDateTimeDayDifference = zonedDateTimeDayDifference(startZdt, endZdt);
+    long zonedDateTimeMinuteDifference = zonedDateTimeMinuteDifference(startZdt, endZdt);
+
+    LOGGER.trace(
+        "PERIOD between given time day :{} , minutes :{}",
+        zonedDateTimeDayDifference,
+        zonedDateTimeMinuteDifference);
+
+    if (zonedDateTimeDayDifference < 0 || zonedDateTimeMinuteDifference <= 0) {
```
[the remark this answers is not in the document]
Comment (ankitmashu) [reply src/main/java/iudx/rs/proxy/metering/util/DateValidation.java]: solved

**Review by gopal-mahajan — APPROVED** [review]
LGTM

File: src/main/java/iudx/rs/proxy/apiserver/ApiServerVerticle.java [hunk 2]
```diff
@@ -756,6 +774,85 @@ private void handlePostEntitiesQuery(RoutingContext routingContext) {
...
+    authInfo.put(STARTT, request.getParam(STARTT));
+    authInfo.put(ENDT, request.getParam(ENDT));
+    LOGGER.debug("auth info = " + authInfo);
+    HttpServerResponse response = routingContext.response();
+
+    String iid = authInfo.getString("iid");
+    String role = authInfo.getString("role");
+
+    if (!VALIDATION_ID_PATTERN.matcher(iid).matches()
+        && (role.equalsIgnoreCase("provider") || role.equalsIgnoreCase("delegate"))) {
```
[the remark this answers is not in the document]
Comment (ankitmashu) [reply src/main/java/iudx/rs/proxy/apiserver/ApiServerVerticle.java:825]: this check is for open provider token

**Review by gopal-mahajan — APPROVED** [review]
LGTM

**Review by ananjaykumar2 — APPROVED** [review]
LGTM

**Pull request merged**